build/commands/scripts/lint.py

```diff
@@ -109,8 +107,8 @@ def main(args):
     # Run format checks
     upstream_branch = cl.GetUpstreamBranch()
```

Comment:
this could alll be replaced by `format_output = RunFormatCheck(base_branch or cl.GetUpstreamBranch())` i think?